backend/tomato/api/elements.py

```diff
@@ -299,6 +299,28 @@ def element_usage(id): #@ReservedAssignment
 	el = _getElement(id)
 	return el.totalUsage.info()	
 
+def element_migrate(id,host): #@ReservedAssignment
```

Comment:
You do not need this API method. Migrate is defined as an action so you can call element_action(ID, "migrate", ...)